operators/bosh-restore-operator/BoshRestoreService.js

```diff
@@ -522,7 +524,13 @@ class BoshRestoreService extends BaseDirectorService {
       _.set(patchResourceObj, 'status.response', patchObj);
     }
     await eventmesh.apiServerClient.patchResource(patchResourceObj);
-    return this.patchRestoreFileWithFinalResult(resourceOptions, patchObj);
+    await this.patchRestoreFileWithFinalResult(resourceOptions, patchObj);
+    if (this.service.pitr === true) {
+      this.reScheduleBackup({
```

Comment:
@amitmalav modified flow to schedule backup after the state is updated in CRD and restoreFile. Also as the `reScheduleBackup` call is not being awaited, it's failure won't affect restore flow, similar to old restore flow. Please have a look once.